hack/unwanted-dependencies.json

```diff
@@ -180,21 +148,51 @@
       "go.opencensus.io": [
         "github.com/Microsoft/hcsshim"
       ],
+      "google.golang.org/appengine": [
+        "github.com/prometheus/client_golang"
+      ],
       "golang.org/x/exp": [
+        "github.com/Microsoft/hcsshim",
         "github.com/antlr4-go/antlr/v4",
+        "github.com/cilium/ebpf",
```

Comment:
Expanding deps on an explicitly unstable module is *extremely* unfortunate. We previously refused to bump to a cilium/epbf version that added this dependency in https://github.com/kubernetes/kubernetes/pull/117457#issuecomment-1632974084, asked epbf to drop this dep in https://github.com/cilium/ebpf/issues/1095, and they refused.